libraries/botbuilder-lg/src/templates.ts

```diff
@@ -268,20 +270,32 @@ export class Templates implements Iterable<Template> {
     */
     public updateTemplate(templateName: string, newTemplateName: string, parameters: string[], templateBody: string): Templates {
         const template: Template = this.items.find((u: Template): boolean => u.name === templateName);
-        if (template === undefined) {
-            return this;
+        if (template !== undefined) {
```

Comment:
>template !== undefined [](start = 12, length = 22)

Would if (template) make sense or do you need to do the next steps even if template is an empty string?

Reply:
#Solved